internal/controller/vrg_volsync.go

```diff
@@ -10,12 +10,17 @@ import (
 
 	"github.com/go-logr/logr"
 	ramendrv1alpha1 "github.com/ramendr/ramen/api/v1alpha1"
+	"github.com/ramendr/ramen/internal/controller/cephfscg"
 	"github.com/ramendr/ramen/internal/controller/util"
 	"github.com/ramendr/ramen/internal/controller/volsync"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// ConsistencyGroupLabel (TODO) use ConsistencyGroupLabel in PR https://github.com/RamenDR/ramen/pull/1472/files
+var ConsistencyGroupLabel = "ramendr.openshift.io/consistency-group"
```

Comment:
We will use the same label found in this PR https://github.com/RamenDR/ramen/pull/1472 once merged.

Reply:
yes, it will be changed once PR 1472 merged